client/src/AppBundle/Service/Client/Sirius/SiriusDocumentUpload.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+
+namespace AppBundle\Service\Client\Sirius;
```

Comment:
I think this should be in `AppBundle\Model\Sirius` since it's a model, not a service (same with the others)

Reply:
Agree! Moving over now.